Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/BoxComponentMode.cpp

```diff
@@ -18,10 +18,10 @@ namespace AzToolsFramework
     }
 
     BoxComponentMode::BoxComponentMode(
-        const AZ::EntityComponentIdPair& entityComponentIdPair, const AZ::Uuid componentType)
+        const AZ::EntityComponentIdPair& entityComponentIdPair, const AZ::Uuid componentType, bool allowAsymmetricalEditing)
         : EditorBaseComponentMode(entityComponentIdPair, componentType)
     {
-        m_boxEdit.Setup(entityComponentIdPair);
+        m_boxEdit.Setup(entityComponentIdPair, allowAsymmetricalEditing);
```

Comment:
suggestion: Might it be nice to have this be an enum instead to make the callsite clearer, so it might be...

```
enum class BoxShapeManipulatorMode
{
    Symmetric,
    Asymmetric 
};
```

Reply:
Once the feature flag is removed, all the existing uses of box manipulators will allow asymmetrical editing, so this may not be needed at all.

Reply:
Okay sounds good! 👍